### ISSUE TITLE
Serde logical types

### ISSUE THREAD

**Issue by ritchie46** [pola-rs/polars#2021]
Date/Datetime/Time are represented by integers. With `SERDE` serialization the physical types, (the integers are serialized), but this should be `chrono` types/ strptime strings.

**Comment by orlp** [comment]
I don't think we want to pin our core data serialization formats to specific third party crates through serde.
